optaplanner-build-parent/pom.xml

```diff
@@ -1014,6 +1017,18 @@
         <groupId>net.revelc.code</groupId>
         <artifactId>impsort-maven-plugin</artifactId>
       </plugin>
+      <plugin> <!-- Make sure the build fails-fast on Javadoc issues. -->
```

Comment:
Javadoc now runs in every module. Fails fast, as opposed to once every blue moon when `optaplanner-distribution` is being built.

Reply:
(CI does do -Dfull always.)

Anyway, why not, let's see how this impacts speed build. I am happy to see spotbugs and checkstyle go for build speedup :)